docs/concepts/conda_pypi.md

```diff
@@ -0,0 +1,102 @@
+Pixi is built on top of both the conda and PyPI ecosystems.
+
+**Conda** is a cross-platform, cross-language package ecosystem that allows users to install packages and manage environments.
+It is widely used in the data science and machine learning community, but it is also used in other fields.
+It’s power comes from the fact that it always installs binary packages, meaning that it doesn’t need to compile anything.
+This makes the ecosystem very fast and easy to use.
+
+**PyPI** is the Python Package Index, which is the main package index for Python packages.
+It is a much larger ecosystem than conda, especially because the boundary to upload packages is lower.
+This means that there are a lot of packages available, but it also means that the quality of the packages is not always as high as in the conda ecosystem.
+
+Pixi **can** install packages from **both** **ecosystems**, but it uses a **conda-first approach**.
+
+The simplified process is as follows:
+
+1. Resolve the conda dependencies.
+2. Map the conda packages to PyPI packages.
+3. Resolve the [remaining]("All pypi dependencies already installed as conda dependencies will not be fetched from PyPI") PyPI dependencies.
+
+
+## Tool Comparison
+Here is a non-exhaustive comparison of the features of conda and PyPI ecosystems.
+
+| Feature | Conda | PyPI |
+| ------- | ----- | ---- |
+| Package format | Binary | Source & Binary (wheel) |
+| Package managers | [`conda`](https://github.com/conda/conda), [`mamba`](https://github.com/mamba-org/mamba), [`micromamba`](https://github.com/mamba-org/mamba), [`pixi`](https://github.com/prefix-dev/pixi), [`rattler`](https://github.com/conda/rattler) | [`pip`](https://github.com/pypa/pip), [`poetry`](https://github.com/python-poetry/poetry), [`uv`](https://github.com/astral-sh/uv), [`pdm`](https://github.com/pdm-project/pdm), [`hatch`](https://github.com/pypa/hatch), [`rye`](https://github.com/astral-sh/rye), [`pixi`](https://github.com/prefix-dev/pixi) |
```

Comment:
Rattler isn't exactly a package manager, right?

Also I think it's fair to skip `rye`. That one is basically dead AFAIK